### PR TITLE
Лабораторная№1

### PR DESCRIPTION
**Pull request by Tatuana4444**

### CODE REVIEW

File: lab1_1.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
```
Comment (Tatuana4444) [inline lab1_1.html:3]: https://www.w3schools.com/HTML/html_responsive.asp

File: lab1_1.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+<html>
+  <head>
+    <title>Бондаревой Татьяны гр. 751001</title>
+    <link rel="stylesheet" href="style.css" >
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous" >
+  </head>
+  <body>
+   <header>
+   	 <nav class="topmenu">    	
+    	<p>Navigation</p>
```
Comment (Tatuana4444) [inline lab1_1.html]: h1 h2 h3 ....

File: lab1_1.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    	<img src="picture/NavBlack.png" class ="nav_img_on_text" >
+    	<input type="checkbox" name="toggle" id="menu" class="toggle_menu">
```
Comment (Tatuana4444) [inline lab1_1.html]: recommend : button / a

File: lab1_1.html [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    	<label for="menu" class="toggle_menu"><i class="fas fa-bars"></i></label>    	
+    	<ul>
+    		<li><a href="">PSD WEBSITES</a></li>
```
Comment (Tatuana4444) [inline lab1_1.html]: https://stackoverflow.com/questions/11078509/how-to-increase-the-clickable-area-of-a-a-tag-button

File: lab1_1.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    		<li><a href="">UI KITS</a></li>
```
Comment (Tatuana4444) [inline lab1_1.html]: # instead of ''

File: lab1_1.html [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    		<li><a href="">PREMIUM PSD FILES</a></li>
+    		<li><a href="">ABOUT</a></li>
+    		<li><a href="">BLOG</a></li>
+    	</ul>
+	 </nav>
+	</header>
+	<section>
+		<img src="picture/Banner.png" class ="banner">
+		<a href=""><i class="fas fa-angle-right arrow_banner"></i></a>
+		<p class="maintext">Awesome Free PSD templates & Resources</p>
```
Comment (Tatuana4444) [inline lab1_1.html]: h1 h2

File: lab1_1.html [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+		<p class="linktosite"><a href=""></a>www.BlazRobar.com</p>
```
Comment (Tatuana4444) [inline lab1_1.html]: h3 h4

File: style.css [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+.pic3{
+	position: absolute;
+	width: 34%;
+	padding-top:15vw;
+	padding-left: 66%;
+	left: 0px;
+}
+
+.underbanner{
+    position: absolute;
```
Comment (Tatuana4444) [inline style.css:109]: how it works

**Comment by aliaksandrkrasnou** [comment]
Approved, nice 👍

**Pull request closed without merging**